.devcontainer/Dockerfile

```
@@ -1,4 +1,4 @@
-ARG VARIANT="1.21"
+ARG VARIANT="1.22.7"
```

we don't need to overspecify this to a patch version
```suggestion
ARG VARIANT="1.22"
```